dissect/target/helpers/configutil.py

```diff
@@ -20,6 +20,8 @@
     Union,
 )
 
+from defusedxml import ElementTree
```

Comment:
Incorporating the `_tree()` function below requires `defaultdict`. should be imported somewhere around here.

Reply:
added